cegs_portal/search/templates/search/v1/partials/_features.html

```diff
@@ -13,7 +12,7 @@
                 </svg>
                 Genome Features
             </div>
-            <div class="justify-self-end">
+            <div style="position: absolute; right: 0;">
```

Comment:
You should use the appropriate tailwind classes when you can, instead of putting CSS in the style tag.